cmd/gateway-main.go

```diff
@@ -209,6 +209,7 @@ func gatewayMain(ctx *cli.Context) {
 	endpointAddr := ctx.Args().Get(1)
 	// Third argument is the address flag
 	serverAddr := ctx.String("address")
+	fatalIf(CheckLocalServerAddr(serverAddr), "Invalid address ‘%s’ in command line argument.", serverAddr)
```

Comment:
@harshavardhana, are we merging GCS branch onto master before the next release? If so I can close this PR.